proxy_docker/app/script/confirmation.sh

```diff
@@ -83,27 +74,28 @@ confirmation() {
 
     # Let's first insert the tx in our DB
 
-    local tx_ts_firstseen=$(echo "${tx_details}" | jq '.result.timereceived')
-    local tx_amount=$(echo "${tx_details}" | jq '.result.amount')
+    local tx_ts_firstseen=$(echo "${tx_details}" | jq '.timereceived')
+    local tx_amount=$(echo "${tx_details}" | jq '.amount')
 
     local tx_size=$(echo "${tx_raw_details}" | jq '.result.size')
     local tx_vsize=$(echo "${tx_raw_details}" | jq '.result.vsize')
-    local tx_replaceable=$(echo "${tx_details}" | jq -r '.result."bip125-replaceable"')
+    local tx_replaceable=$(echo "${tx_details}" | jq -r '."bip125-replaceable"')
     tx_replaceable=$([ ${tx_replaceable} = "yes" ] && echo "true" || echo "false")
 
     local fees=$(compute_fees "${txid}")
     trace "[confirmation] fees=${fees}"
+    trace "[confirmation] tx_hash=${tx_hash}"
 
     # If we missed 0-conf...
     local tx_blockhash=null
     local tx_blockheight=null
     local tx_blocktime=null
     if [ "${tx_nb_conf}" -gt "0" ]; then
       trace "[confirmation] tx_nb_conf=${tx_nb_conf}"
-      tx_blockhash="$(echo "${tx_details}" | jq -r '.result.blockhash')"
-      tx_blockheight=$(get_block_info ${tx_blockhash} | jq '.result.height')
+      tx_blockhash="$(echo "${tx_details}" | jq -r '.blockhash')"
+      tx_blockheight=$(get_block_info ${tx_blockhash} | jq '.height')
```

Comment:
We calling the Bitcoin Core container here to get block height... Do you think it's a good idea to do this stuff in the walletnotify.sh script in the Bitcoin container and include it in the message?  Instead of calling it here from the proxy...

Reply:
Hey we have the blockheight and blockhash fields in gettransaction's result!  It's not even needed anymore to call getblockinfo...

Reply:
done